rust-runtime/aws-smithy-runtime-api/src/client/retries.rs

```diff
@@ -44,6 +44,15 @@ impl ShouldAttempt {
             _ => panic!("Expected this to be the `YesAfterDelay` variant but it was the `{self:?}` variant instead"),
         }
     }
+
+    /// If this isn't a `No` variant, panic.
+    pub fn expect_no(self) {
```

Comment:
Is this method used outside tests?

Reply:
No. It's defined in an impl block with `#[cfg(feature = "test-util")]`.